src/bot/mod.rs

```diff
@@ -73,5 +78,9 @@ pub async fn entrypoint(database_pool: Pool<MySql>, redis_client: Client) {
             })
         });
 
-    framework.run().await.expect("Err creating client");
-}
\ No newline at end of file
+    let built_framework = framework.build().await.expect("Err building poise client");
+
+    logging::install_framework(built_framework.clone());
```

Comment:
is there a reason to give the framework like this instead of passing it inside the setup lambda (74-77)?

Reply:
Ok fair enough. Then that is fine